plugins-compat-tester/src/main/java/org/jenkins/tools/test/exception/PomExecutionException.java

```diff
@@ -84,4 +84,13 @@ public String getErrorMessage(){
     public ExecutedTestNamesDetails getTestDetails() {
         return testDetails;
     }
+    
+    public boolean failedOnTestExecution() {
+        for (Throwable throwable : exceptionsThrown) {
+            if (throwable instanceof ExecutedTestNamesSolverException) {
```

Comment:
Does this differentiate between a failed test and a SIGTERM/SIGKILL being sent? If it does not then that means we are heavily changing the PCT behaviour under a legitimate test failure, now it would stop when it is supposed to continue running with the rest of the tests...

Reply:
Yes it does, if there is a failed test, then no problem there, I mean, it will be part of the `TestExecutionResult.getTestDetails().getFailed()` collection.
Actually, this code is evaluated on a catch block for a `PomExecutionException` from the generation of that `TestExecutionResult` bean, so if there is at least on `ExecutedTestNamesSolverException` inside the list of `Throwable` that motivated the `PomExecutionException` it will came from a `ParserConfigurationException`, `SAXException`, `IOException` or a crashed test. For all the cases the PCT should be able to deal with instead of ignoring it (what it is doing nowadays)